config/clusters/qcl/support.values.yaml

```diff
@@ -0,0 +1,35 @@
+prometheusIngressAuthSecret:
+  enabled: true
+
+redirects:
+  rules:
+    - from: staging.qcl.2i2c.cloud
+      to: staging.quantifiedcarbon.com
```

Comment:
Nice work figuring this out! I wasn't aware of this system to create Ingress resources was available in the `basehub` helm chart, and suspect it may not have been documented either yet.

We should have redirect rules for grafana and the production hub here as well. Maybe the production hub is out of scope for this PR and thats fine.

Reply:
There is some documentation here https://infrastructure.2i2c.org/en/latest/howto/manage-domains/index.html

I think when we start work on the deployer improvments making the generator take optional args for custom domains might help

Yes I think this is good to go and I'll split out the prod hub.